integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkEnvIT.java

```diff
@@ -90,23 +94,27 @@ private void initHiveEnv() {
             "thrift://%s:%d",
             containerSuite.getHiveContainer().getContainerIpAddress(),
             HiveContainer.HIVE_METASTORE_PORT);
+    warehouse =
+        String.format(
+            "hdfs://%s:%d/user/hive/warehouse",
+            containerSuite.getHiveContainer().getContainerIpAddress(),
+            HiveContainer.HDFS_DEFAULTFS_PORT);
   }
 
   private void initSparkEnv() {
-    sparkSession =
-        SparkSession.builder()
-            .master("local[1]")
-            .appName("Spark connector integration test")
-            .config("spark.plugins", GravitinoSparkPlugin.class.getName())
-            .config(GravitinoSparkConfig.GRAVITINO_URI, gravitinoUri)
-            .config(GravitinoSparkConfig.GRAVITINO_METALAKE, metalakeName)
-            .config(
-                "spark.sql.warehouse.dir",
-                String.format(
-                    "hdfs://%s:%d/user/hive/warehouse",
-                    containerSuite.getHiveContainer().getContainerIpAddress(),
-                    HiveContainer.HDFS_DEFAULTFS_PORT))
-            .enableHiveSupport()
-            .getOrCreate();
-  }
+    if (sparkSession != null) {
+      sparkSession.close();
```

Comment:
Add a check for sparksession because of the following error

```
SparkIcebergCatalogIT > initializationError FAILED
    java.lang.IllegalStateException: Should not create duplicate GravitinoCatalogManager
        at com.google.common.base.Preconditions.checkState(Preconditions.java:512)
        at com.datastrato.gravitino.spark.connector.catalog.GravitinoCatalogManager.create(GravitinoCatalogManager.java:43)
        at com.datastrato.gravitino.spark.connector.plugin.GravitinoDriverPlugin.init(GravitinoDriverPlugin.java:46)
        at org.apache.spark.internal.plugin.DriverPluginContainer.$anonfun$driverPlugins$1(PluginContainer.scala:53)
        at scala.collection.TraversableLike.$anonfun$flatMap$1(TraversableLike.scala:293)
        at scala.collection.mutable.ResizableArray.foreach(ResizableArray.scala:62)
        at scala.collection.mutable.ResizableArray.foreach$(ResizableArray.scala:55)
        at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:49)
        at scala.collection.TraversableLike.flatMap(TraversableLike.scala:293)
        at scala.collection.TraversableLike.flatMap$(TraversableLike.scala:290)
        at scala.collection.AbstractTraversable.flatMap(Traversable.scala:108)
        at org.apache.spark.internal.plugin.DriverPluginContainer.<init>(PluginContainer.scala:46)
        at org.apache.spark.internal.plugin.PluginContainer$.apply(PluginContainer.scala:210)
        at org.apache.spark.internal.plugin.PluginContainer$.apply(PluginContainer.scala:193)
        at org.apache.spark.SparkContext.<init>(SparkContext.scala:565)

```